src/ol/interaction/pinchzoom.js

```diff
@@ -29,6 +29,13 @@ ol.interaction.PinchZoom = function(opt_options) {
 
   /**
    * @private
+   * @type {boolean}
+   */
+  this.constrainResolution_ = options.constrainResolution ?
+options.constrainResolution : false;
```

Comment:
Alternatively, to avoid repeating the variable as the conditional and consequent, this could be:
```js
this.constrainResolution_ = options.constrainResolution || false;
```